### PR TITLE
Removed "warm migration"

### PR DESCRIPTION
**Pull request by apinnick**
No BZ or Jira.

Removed "warm migration" because that does not exist in MTC.

4.6+

Preview: https://deploy-preview-35936--osdocs.netlify.app/openshift-enterprise/latest/migration_toolkit_for_containers/troubleshooting-mtc.html#migration-viewing-migration-plan-log_troubleshooting-mtc

No QE required. Peer review needed.

### CODE REVIEW

File: modules/migration-viewing-migration-plan-log.adoc [hunk 1]
```diff
@@ -22,7 +22,7 @@ The command displays the filtered logs of the following pods:
 . In the {mtc-short} web console, click *Migration Plans*.
 . Click the *Migrations* number next to a migration plan to view the *Migrations* page.
 +
-The *Migrations* page displays the migration types associated with the migration plan, for example, *Stage* or *Cutover* for warm migration.
+The *Migrations* page displays the migrations associated with the migration plan, for example, *Stage* or *Cutover*.
```
Comment (mikemckiernan) [inline modules/migration-viewing-migration-plan-log.adoc:25]: It's nitpicky, but it feels like a run-on sentence.  Obligatory sugg: "...migration plan: *Stage* or *Cutover*."

**Review by mikemckiernan — COMMENTED** [review]
Up-to-you nit.

File: modules/migration-viewing-migration-plan-log.adoc (hunk 1 above)
Comment (apinnick) [reply modules/migration-viewing-migration-plan-log.adoc:25]: Actually, now I can update this sentence with the actual text (Stage, Cutover, Rollback) and remove "for example).

Comment (apinnick) [reply modules/migration-viewing-migration-plan-log.adoc:25]: In fact, that whole sentence is pretty much irrelevant, so I am going to remove it.

**Pull request merged**